### PR TITLE
Add "How to Start" Section into README.md

### PR DESCRIPTION
**Pull request by PuHsiu**
Add some content to "How to Start"

### CODE REVIEW

**Comment by sugyan** [comment]
@kenneth-lau please review this.

File: README.md [hunk 1]
```diff
@@ -42,6 +42,81 @@ func main() {
...
+```
+
+With reply token, you can reply message via ```ReplyMessage()```
+
+```go
+    var messages []linebot.Message
+
+    // append some message to messages
+
+    _, err := bot.PushMessage( replyToken, messages... ).Do()
```
Comment (sugyan) [inline README.md]: please remove needless whitespace.

File: README.md [hunk 2]
```diff
@@ -42,6 +42,81 @@ func main() {
 	...
 ```
 
+## How to Start ##
+
+Line Messaging API use JSON to format data.
+```ParseRequest()``` will help you to parse the ```*http.Request``` content and return a slice of Event Object.
+
+```go
+    events := bot.ParseRequest(req)
```
Comment (mokejp) [inline README.md]: `ParseRequest` returns `[]*Event` and `error`.
So you have to add `err` in left side.

File: README.md [hunk 3]
```diff
@@ -42,6 +42,85 @@ func main() {
 	...
 ```
 
+## How to Start ##
+
```
Comment (kenneth-lau) [inline README.md]: How to start

File: README.md [hunk 4]
```diff
@@ -42,6 +42,85 @@ func main() {
...
+Line Messaging API use JSON to format data.
```
Comment (kenneth-lau) [inline README.md]: The LINE Messaging API uses the JSON data format.

File: README.md [hunk 5]
```diff
@@ -42,6 +42,85 @@ func main() {
...
+	for _, event := range events {
+		if event.Type == linebot.EventTypeMessage {
+			// Do Something...
+		}
+	}
+```
+
+### Receiver ###
+
+To send a message to user/group/room, you might need an ID
```
Comment (kenneth-lau) [inline README.md]: To send a message to a user, group, or room, you need either an ID

File: README.md [hunk 6]
```diff
@@ -42,6 +42,85 @@ func main() {
...
+
+```go
+	userID := event.Source.UserID
+	groupID := event.Source.GroupID
+	RoomID := event.Source.RoomID
+```
+
+or reply token
```
Comment (kenneth-lau) [inline README.md]: or a reply token.

File: README.md [hunk 7]
```diff
@@ -42,6 +42,85 @@ func main() {
...
+
+```go
+	replyToken := event.ReplyToken
+```
+
+### Create Message ###
```
Comment (kenneth-lau) [inline README.md]: Create message

File: README.md [hunk 8]
```diff
@@ -42,6 +42,85 @@ func main() {
...
+
+Line Messaging API supply many types of message, just use the ```New<Type>Message()``` to create it.
```
Comment (kenneth-lau) [inline README.md]: The LINE Messaging API provides various types of messages. To create a message use ```New<Type>Message()```.

File: README.md [hunk 9]
```diff
@@ -42,6 +42,85 @@ func main() {
...
+```go
+	leftBtn := linebot.NewMessageTemplateAction("left", "left clicked")
+	rightBtn := linebot.NewMessageTemplateAction("right", "right clicked")
+
+	template := linebot.NewConfirmTemplate("Hello World", leftBtn, rightBtn)
+
+	messgage := linebot.NewTemplateMessage("Sorry :(, please update your app.", template)
+```
+
+### Send Message ###
```
Comment (kenneth-lau) [inline README.md]: Send message

File: README.md [hunk 10]
```diff
@@ -42,6 +42,85 @@ func main() {
...
+
+With ID, you can send message via ```PushMessage```
```
Comment (kenneth-lau) [inline README.md]: With an ID, you can send messages using ```PushMessage```.

File: README.md [hunk 11]
```diff
@@ -42,6 +42,85 @@ func main() {
...
+
+	// append some message to messages
+
+	_, err := bot.PushMessage(ID, messages...).Do()
+	if err != nil {
+		// Do something when some bad happened
+	}
+```
+
+With reply token, you can reply message via ```ReplyMessage()```
```
Comment (kenneth-lau) [inline README.md]: With a reply token, you can reply to messages using ```ReplyMessage()```.

**Comment by sugyan** [comment]
Thank you very much!

**Pull request merged**